### DISCUSSION TITLE
1.0.2

### DISCUSSION THREAD

**Discussion by cswendrowski** [cswendrowski/FoundryVTT-vox-ludos-speech#4] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/cswendrowski/FoundryVTT-vox-ludos-speech/releases/tag/1.0.2'>1.0.2</a>.</em>

**Comment by hmqgg** [comment]
Currently, VoxLudos doesn't publish a release named `latest` so that checking update will pop out a 404 error.

It suggests using `.../releases/latest/download/module.json` which comes from a GitHub feature instead of `.../releases/download/latest/module.json` that requires a release named `latest` and continuously update.
